src/order/category/ωCPO.lean

```diff
@@ -0,0 +1,269 @@
+/-
```

Comment:
Are we tempting fate having a unicode file name? I'm tempted to rename this `omega_complete_partial_order.lean`. It feels more likely that someone will have a setup that gets confused by the unicode path than that the short name will matter.

Reply:
It will also match the one in `order`.